protocol-dashboard/src/components/UpdateDelegationModal/UpdateDelegationModal.tsx

```diff
@@ -398,20 +401,29 @@ const UpdateDelegationModal: React.FC<UpdateDelegationModalProps> = ({
 type ManageDelegationProps = {
   delegates: BN
   wallet: Address
+  disabledReason?: string | null
 }
 export const ManageDelegation = ({
   delegates,
-  wallet
+  wallet,
+  disabledReason
 }: ManageDelegationProps) => {
   const { isOpen, onClick, onClose } = useModalControls()
   return (
     <>
-      <Button
-        type={ButtonType.PRIMARY}
-        text={messages.manage}
-        css={{ width: '100%' }}
-        onClick={onClick}
-      />
+      <BasicTooltip
+        position={Position.TOP}
+        text={disabledReason}
+        isDisabled={!Boolean(disabledReason)}
+      >
+        <Button
+          type={ButtonType.PRIMARY}
+          text={messages.manage}
+          css={{ width: '100%' }}
```

Comment:
use fullWidth prop unless you want the expand animation

Reply:
unfort this is the stems button which has no `fullWidth` 🥲🥲🥲🥲🥲🥲🥲🥲